app/views/ExposureHistory/NextSteps.tsx

```diff
@@ -32,22 +38,23 @@ const NextSteps = (): JSX.Element => {
   const buttonText = t('exposure_history.next_steps.button_text');
 
   const handleOnPressTakeAssessment = () => {
-    navigation.navigate(Screens.SelfAssessment);
+    Linking.openURL(AUTHORITY_ADVICE_URL);
```

Comment:
what do you think bout something like: 

```suggestion
    AUTHORITY_ADVICE_URL ? Linking.openURL(AUTHORITY_ADVICE_URL) : navigation.navigate(Screens.SelfAssessment)
```

its not clear to me that every HA will have an adviceUrl and it might be nice to leave ourselves open to this case

Reply:
Yep, that seems right. Any thoughts about redirecting to the self-assessment screen when HA doesn't have an advice URL or at least disabling the button?